snmp/datadog_checks/snmp/data/profiles/_base_hp.yaml

```diff
@@ -0,0 +1,190 @@
+    # NOTE: other(1), notAvailable(2), disabled(3), enabled(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.1
+      name: cpqHeAsrStatus
+
+  - # Whether the ASR timer passed the server Power-On Self-Test (POST).
+    # NOTE: other(1), failed(2), ok(3)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.6
+      name: cpqHeAsrPost
+
+  - # Number of ASR reboots that occurred on the server since last manual reboot.
+    # NOTE: -1 if not supported.
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.10
+      name: cpqHeAsrRebootCount
+    forced_type: monotonic_count
+
+  - # Overall condition of the ASR feature.
+    # NOTE: other(1), notAvailable(2), disabled(3), enabled(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.17
+      name: cpqHeAsrCondition
+
+  - # Status of the ASR network access feature.
+    # NOTE: other(1), disabled(2), enabled(3)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.21
+      name: cpqHeAsrNetworkAccessStatus
+
+  # Temperature and fans.
+
+  - # Status of temperature sensors.
+    # NOTE: other(1), ok(2), degraded(3), failed(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.6.3
+      name: cpqHeThermalTempStatus
+
+  - # Status of fan(s).
+    # NOTE: other(1), ok(2), degraded(3), failed(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.6.4
+      name: cpqHeThermalSystemFanStatus
+
+  - # Status of processor fan(s).
+    # NOTE: other(1), ok(2), failed(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.6.5
+      name: cpqHeThermalCpuFanStatus
+
+  - # Temperator sensors.
+    MIB: CPQHLTH-MIB
+    table:
+      OID: 1.3.6.1.4.1.232.6.2.6.8
+      name: cpqHeTemperatureTable
+    symbols:
+      - OID: 1.3.6.1.4.1.232.6.2.6.8.1.4
+        name: cpqHeTemperatureCelsius
+      - # NOTE: other(1), ok(2), degraded(3), failed(4)
+        OID: 1.3.6.1.4.1.232.6.2.6.8.1.6
+        name: cpqHeTemperatureCondition
+    metric_tags:
+      - tag: temperature_index
+        column:
+          OID: 1.3.6.1.4.1.232.6.2.6.8.1.3
+          name: cpqHeTemperatureIndex
+
+  # Power supplies.
+
+  - # Current power meter reading in Watts.
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.15.3
+      name: cpqHePowerMeterCurrReading
+
+  - # A table of battery entries.
+    MIB: CPQHLTH-MIB
+    table:
+      OID: 1.3.6.1.4.1.232.6.2.17.2
+      name: cpqHeSysBatteryTable
+    symbols:
+      - # NOTE: other(1), ok(2), degraded(3), failed(4)
+        OID: 1.3.6.1.4.1.232.6.2.17.2.1.4
+        name: cpqHeSysBatteryCondition
+      - # NOTE: 15 different statuses are possible -- not listed here, see MIB.
+        OID: 1.3.6.1.4.1.232.6.2.17.2.1.5
+        name: cpqHeSysBatteryStatus
+    metric_tags:
+      - tag: battery_index
+        column:
+          OID: 1.3.6.1.4.1.232.6.2.17.2.1.2
+          name: cpqHeSysBatteryIndex
```

Comment:
Is this the right MIB? I can't find the cpqHeSysBatteryTable or metrics here http://www.circitor.fr/Mibs/Html/C/CPQHLTH-MIB.php

Reply:
Looking back at https://github.com/DataDog/integrations-core/pull/5637, it was found here:   https://support.hpe.com/hpsc/swd/public/detail?swItemId=MTX_53293d026fb147958b223069b6 which seems to have a more recent mib file than circitor